packages/material-ui/src/Slider/Slider.js

```diff
@@ -683,25 +679,38 @@ const Slider = React.forwardRef(function Slider(props, ref) {
     doc.addEventListener('touchend', handleTouchEnd);
   });
 
+  const handleMouseMove = useEventCallback((event) => {
+    // require the left mouse button still be held
+    if (event.buttons === 1) {
+      handleTouchMove(event);
+    } else {
+      touchId.current = undefined;
+      stopListening();
```

Comment:
Shouldn't we call `handleTouchEnd()` instead to clear the active and open states?

Reply:
That was my first thought too. It doesn't have a proper `finger` in this scenario so it short circuits and doesn't run any of the cleanup code, that was why I finally decided to DRY the cleanup so it's easier to call from here.

Reply:
Why wouldn't it have a proper `finger`? We use the mouse position when not a touch event.